### PR TITLE
Include django-filter in the kolibri-static.

### LINKED ISSUE #218: AttributeError: `Notetype` object has no attribute `MethodFilter`
## Summary
I generated a `kolibri-static`via `python setup.py bdist_wheel --static` then install it on my `Windows 7 VM` using pip then run `kolibre manage runserver` 
I got an error see the screenshots below.

## Branch or installer
 - Version: `master`

## Screenshots
screen shot 2016-07-09 at 2 34 43 am

## How to reproduce
1. cd ~/kolibri in your working machine
2. run command `python setup.py bdist_wheel --static`
3. install the kolibri-static in your Windows 7 virtualbox

### PR DESCRIPTION
**Pull request by arceduardvincent**
## Summary
Hi @aronasorman and @cpauya 
* I Include the django-filter in the kolibri-static. 
* Update the `Makefile` to create a `kolibri-static`
* Able to install the `kolibri-static` in Windows 7 VM

List the issues solved or partly solved by the PR
Fixes #218

## Screenshots 
Successfully install the `kolibri-static` and run command `kolibri manage runserver`
screen shot 2016-07-09 at 3 48 10 am

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def read_file(fname):
 install_requires = [
     'colorlog',
     'django>=1.9,<1.10',
+    'django-filter==0.13.0',
```
Comment (aronasorman) [inline setup.py]: Can you make this a greater than comparison? Just so we don't have to keep changing this in the future.

Comment (aronasorman) [inline setup.py]: Actually nvm.

File: MANIFEST.in [hunk 2]
```diff
@@ -3,15 +3,7 @@ include CONTRIBUTING.rst
...
-recursive-include kolibri/auth *
-recursive-include kolibri/content *
-recursive-include kolibri/core *
-recursive-include kolibri/deployment *
-recursive-include kolibri/dist *
-recursive-include kolibri/logger *
-recursive-include kolibri/plugins *
-recursive-include kolibri/test *
-recursive-include kolibri/utils *
+recursive-include kolibri/* *
```
Comment (aronasorman) [inline MANIFEST.in]: Tests seem to be failing now. Could it be because we're too eager with our includes in Kolibri?

File: setup.py (hunk 1 above)
Comment (jamalex) [inline setup.py]: Yeah, that would defeat the "pinning" -- despite the goal of backwards compatibility with semantic versioning.... doesn't always happen.

**Comment by arceduardvincent** [comment]
Hi @aronasorman and @jamalex done working with your request above.
As of now if we want to build the `kolibri-static` we should run `python setup.py bdist_wheel --static` for our offline version. I omit the `python setup.py bdist_wheel --static` in the `make sdist` command because it is failing in our travis.

Distribution and installers reference:
http://kolibri.readthedocs.io/en/latest/dev/building.html

**Pull request merged**